RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZ_vect.cc

```diff
@@ -37,16 +37,49 @@ DAClusterizerInZ_vect::DAClusterizerInZ_vect(const edm::ParameterSet& conf) {
   }
   d0CutOff_ = conf.getParameter<double> ("d0CutOff");
   dzCutOff_ = conf.getParameter<double> ("dzCutOff");
-  maxIterations_ = 100;
+  uniquetrkweight_ = conf.getParameter<double>("uniquetrkweight");
+  zmerge_ = conf.getParameter<double>("zmerge");
+
+  if(verbose_){
+    std::cout << "DAClusterizerinZ_vect: mintrkweight = " << mintrkweight_ << std::endl;
+    std::cout << "DAClusterizerinZ_vect: uniquetrkweight = " << uniquetrkweight_ << std::endl;
+    std::cout << "DAClusterizerinZ_vect: zmerge = " << zmerge_ << std::endl;
+    std::cout << "DAClusterizerinZ_vect: Tmin = " << Tmin << std::endl;
+    std::cout << "DAClusterizerinZ_vect: Tpurge = " << Tpurge << std::endl;
+    std::cout << "DAClusterizerinZ_vect: Tstop = " << Tstop << std::endl;
+    std::cout << "DAClusterizerinZ_vect: vertexSize = " << vertexSize_ << std::endl;
+    std::cout << "DAClusterizerinZ_vect: coolingFactor = " << coolingFactor_ << std::endl;
+    std::cout << "DAClusterizerinZ_vect: d0CutOff = " << d0CutOff_ << std::endl;
+    std::cout << "DAClusterizerinZ_vect: dzCutOff = " << dzCutOff_ << std::endl;
+  }
+
+
   if (Tmin == 0) {
-    LogDebug("DAClusterizerinZ_vectorized")  << "DAClusterizerInZ: invalid Tmin" << Tmin
-					     << "  reset do default " << 1. / betamax_ << endl;
+    std::cout  << "DAClusterizerInZ: invalid Tmin" << Tmin
+	       << "  reset do default " << 1. / betamax_ << endl;
```

Comment:
This code is executed when nonsensical input values were given. Would a LogWarning be appropriate?